internal/mage/engine.go

```diff
@@ -101,7 +101,7 @@ func (t Engine) Publish(ctx context.Context, version string) error {
 		time.Sleep(3 * time.Second) // allow buildkit logs to flush, to minimize potential confusion with interleaving
 		fmt.Println("PUBLISHED IMAGE REF:", digest)
 
-		return nil
+		return Cli{}.Publish(ctx)
```

Comment:
Keeping this [cohesive](https://www.educative.io/answers/software-engineering-principle-coupling-and-cohesion) & running `mage engine:publish cli:publish` instead seems more aligned with the [Unix philosophy](https://en.wikipedia.org//wiki/Unix_philosophy) to me.